### ISSUE TITLE
[v0.2.2] Release Tracker

### ISSUE THREAD

**Issue by WoosukKwon** [vllm-project/vllm#1551]
**ETA**: ~~Nov 3rd (Fri) - Nov 6th (Mon).~~ Nov 17th (Fri) - 19th (Sun).

## Major changes

* Extensive refactoring for better tensor parallelism & quantization support
* Changes in scheduler: from 1D flattened input tensor to 2D tensor
* Bump up to PyTorch v2.1 + CUDA 12.1
* New models: Yi, ChatGLM, Phi
* Added LogitsProcessor API
* Preliminary support for SqueezeLLM

## PRs to be merged before the release

- [x] CUDA 12 #1527
- [x] Yarn #1264, #1161
- [x] Phi model #1664 
- ~~[ ] Support embedding inputs #1265~~

**Comment by WoosukKwon** [comment]
@zhuohan123 @simon-mo Please feel free to add if you have any PR that should be merged for the next release.

**Comment by simon-mo** [comment]
I can release the corresponding docker image as well! Hopefully we can get logits processor in as well?

**Comment by esmeetu** [comment]
It seems that there is something wrong with batch processing at the current main branch.
When i open a api server that serving wizardcoder-1B model, and testing humaneval by using batch requests and greedy sampling, it gives me less than 10% on main branch whereas 0.2.1-post1 gives me 23.17%.
related issue: #1570
Hope this will be addressed at the v2.2.0. Thanks!

**Comment by esmeetu** [comment]
> It seems that there is something wrong with batch processing at the current main branch.
> When i open a api server that serving https://huggingface.co/WizardLM/WizardCoder-1B-V1.0 model, and testing humaneval by using batch requests(164 concurrent reqs) and greedy sampling, it gives me less than 10% on main branch whereas 0.2.1-post1 gives me 23.17%.
> related issue: #1570
> I have no idea about this and hope this will be addressed at the  coming v0.2.2. Thanks!

#1546 Fix this.

**Comment by incrementallearning** [comment]
When is the v0.2.2 version scheduled to be released?

**Comment by WoosukKwon** [comment]
@incrementallearning It's in the process. We are planning to release it asap. ETA is Nov 17th (Fri) - 19th (Sun).
